src/test/java/net/openhft/chronicle/jlbh/JLBHTest.java

```diff
@@ -83,6 +82,13 @@ public void shouldWriteResultToTheOutputProvided() {
         final String actual = withoutNonDeterministicFields(result);
         System.out.println("actual = " + actual);
 
+        if (!expected.equals(actual)) {
+            System.err.println("expected");
+            expected.chars().limit(10).boxed().forEach(System.err::println);
+            System.err.println("actual");
+            actual.chars().limit(10).boxed().forEach(System.err::println);
+        }
+
         assertEquals(expected, actual);
```

Comment:
why investigate please can we make it so that we don't fail buildAll every time this is run! Please just log, and don't failing ( while investigating )